package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "shortcut-battle",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode production",
+    "lint": "npm-run-all lint:*",
+    "lint:ts": "eslint --ext=.ts,.tsx src",
+    "lint:css": "stylelint src/**/*.css",
+    "prepare": "husky install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MumbaiGamez/shortcut-battle.git"
+  },
+  "keywords": [],
+  "author": "MumbaiGamez",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/MumbaiGamez/shortcut-battle/issues"
+  },
+  "homepage": "https://github.com/MumbaiGamez/shortcut-battle#readme",
+  "devDependencies": {
+    "@svgr/webpack": "^5.5.0",
```

❗️
Версии нужно запинить
```suggestion
    "@svgr/webpack": "5.5.0",
```

Чтобы больше не натыкаться на это, можно запилить `.npmrc` файлик, где явно указать опцию `save-exact=true`. Почему надо пинить? Потому что код, на который опираются наши проекты, пишут люди, которые могут ошибаться, и нас это затрагивать не должно. В качестве общего правила можно думать об этом так: все манипуляции с версиями используемых зависимостей должны быть намеренными и точными

---

Готово 03e0320df9da90c01741bb4829b9fe0d132d9b63